### ISSUE TITLE
-s parameter removed

### ISSUE THREAD

**Issue by d1nuc0m** [pystardust/ytfzf#447]
I've recently upgraded to the 2.x branch, and noticed that the -s option has been deprecated (the one that when the video ended, brought back to a new search). Only the -l option is available, and it brings back to the search result.
Can I ask why it has been removed and if it will be re-implemented in the future?

**Comment by Euro20179** [comment]
It was removed because it added a lot of complexity, however after some refactoring I merged it in #443
try doing this again:
```sh
git clone https://github.com/pystardust/ytfzf
cd ytfzf
sudo make
```

**Comment by d1nuc0m** [comment]
> ```shell
> git clone https://github.com/pystardust/ytfzf
> ```

Thank you, it actually works
